internal/monitors/kubernetes/cluster/metadata.yaml

```diff
@@ -22,6 +22,22 @@ common:
     resource:
       description: The k8s resource that the quota applies to
   metrics:
+    kubernetes.container_cpu.limit:
+      description: Maximum CPU limit set for the container
```

Comment:
Expand this doc to explain that these is just derived straight from the pod spec and which fields.

Reply:
Added details about metrics